lib/vast/Conversion/ABI/EmitABI.cpp

```diff
@@ -227,7 +227,7 @@ namespace vast
                         op.getLoc(),
                         // Temporal, to avoid verification issues, will be changed once
                         // original func is removed.
-                        "vast.abi" + op.getName().str(),
+                        "vast.abi." + op.getName().str(),
```

Comment:
make this some global named constant `vast::abi_prefix`, it is already on two places